src/lib/components/Create/Step2.svelte

```diff
@@ -1,7 +1,6 @@
 <script lang="ts">
-	import Input from '$lib/shared/Input.svelte';
 	import { createEventDispatcher } from 'svelte';
-	const dispatch = createEventDispatcher();
+	import Input from '../Input.svelte';
```

Comment:
Why not have the imports be relative so you can't break anything when you move the file somewhere?

Reply:
I did this to enforce that all components are in the same directory but besides that I agree with it and am open to revert it.

Reply:
Added a dynamic components path